src/main/java/com/microsoftopentechnologies/windowsazurestorage/helper/AzureStorageAccount.java

```diff
@@ -50,18 +50,25 @@ public static class StorageAccountCredential implements java.io.Serializable {
         private final String storageAccountName;
         private final Secret storageAccountKey;
         private final String blobEndpointURL;
+        private final String cdnEndpointURL;
 
         public StorageAccountCredential(
                 String storageAccountName,
                 String storageKey,
-                String endpointURL) {
+                String blobEndpointURL,
+                String cdnEndpointURL) {
```

Comment:
modifying constructor is risky, I need to check if this is safe

Reply:
thanks - I'll wait till I hear back from you about this